sdk-exporter/common/src/test/scala/org/typelevel/otel4s/sdk/exporter/otlp/autoconfigure/OtlpClientAutoConfigureSuite.scala

```diff
@@ -30,12 +31,15 @@ import scalapb_circe.Printer
 
 import scala.concurrent.duration._
 
-class OtlpClientAutoConfigureSuite extends CatsEffectSuite with SuiteRuntimePlatform {
+class OtlpClientAutoConfigureSuite extends CatsEffectSuite {
 
   import OtlpClientAutoConfigure.Defaults
 
   private sealed trait Payload
 
+  private implicit val fs2Compression: Compression[IO] =
+    fs2.io.compression.fs2ioCompressionForIO
```

Comment:
Can also just be an import:
```suggestion
import fs2.io.compression._
```